src/quo/components/counter/collectible_counter/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo.components.counter.collectible-counter.view
+  (:require
+    [quo.components.counter.collectible-counter.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:value {:optional true} [:maybe [:or :string :int]]]
+      [:status {:optional true} [:maybe :keyword]]
+      [:size {:optional true} [:maybe [:enum :size-32 :size-24]]]
+      [:accessibility-label {:optional true} [:maybe :keyword]]
+      [:theme :schema.common/theme]]]]
+   :any])
+
+(defn- view-internal
+  [{:keys [value _status _size _theme accessibility-label]
```

Comment:
Since we are already using malli, we can remove these underscored vars :+1: 

I used many of these vars in the past, but right now I think there are few places where they provide actual value.